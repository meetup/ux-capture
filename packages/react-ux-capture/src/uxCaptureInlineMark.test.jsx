import React from 'react';
import { shallow } from 'enzyme';
import UXCaptureInlineMark from './UXCaptureInlineMark';

const MOCK_PROPS = {
	mark: 'foo',
};

const renderComponent = (props = MOCK_PROPS) =>
	shallow(<UXCaptureInlineMark {...props} />);

describe('UXCaptureInlineMark', () => {
	it('renders component markup', () => {
		expect(renderComponent()).toMatchSnapshot();
	});
});
