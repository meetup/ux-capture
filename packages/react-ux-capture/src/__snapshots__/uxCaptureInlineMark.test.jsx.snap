// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UXCaptureInlineMark renders component markup 1`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "
	<script>
		if(window.UXCapture) {
			window.UXCapture.mark(\\"foo\\");
		}
	</script>
",
    }
  }
/>
`;
